packages/blockfrost-client/scripts/createVersionSource.js

```diff
@@ -0,0 +1,73 @@
+const fs = require('fs');
```

Comment:
This file should be deleted if taking versions from `cardano-services-client`
```suggestion
```

Reply:
There is no longer such a file since I merged Blockfrost providers into `cardano-services`. 1fa15312637c87fb261e833e09c9cbb4f5e31e56